<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>491</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>File</string>
   </property>
   <widget class="QLineEdit" name="lePath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>391</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnChange">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>491</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Target</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Architecture</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxArchitecture">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>PicoBlaze</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Family</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxFamily">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>kcpsm1cpld</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>kcpsm1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>kcpsm2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>kcpsm3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>kcpsm6</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>491</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Options</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Indentation</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Radix</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Linebreak</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Case</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxIntend">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Tabs</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Spaces</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cbxRadix">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Binary</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Octal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Decimal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hexadecimal</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cbxLineBreak">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>LF</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CRLF</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cbxCase">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Lowercase</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Uppercase</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lblTabSize">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TabSize</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leTabSize">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate symbols</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chckReg">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Reg</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chckCode">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Code</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chckData">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Data</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chckPort">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chckConst">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Const</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
